copy/copy.go

```diff
@@ -126,6 +127,17 @@ type Options struct {
 	// Download layer contents with "nondistributable" media types ("foreign" layers) and translate the layer media type
 	// to not indicate "nondistributable".
 	DownloadForeignLayers bool
+
+	// Contains slice of OptionCompressionVariant, where copy will ensure that ImageListSelection
+	// images in the manifest list will exist with compression specificed in OptionCompressionVariant
+	// while performing copy. Invalid when copying a non-multi-architecture image. That will probably
+	// change in the future.
```

Comment:
Something like
```suggestion
	// Contains slice of OptionCompressionVariant, where copy will ensure that for each platform
	// in the manifest list, a variant with the requested compression will exist.
	//
	// Invalid when copying a non-multi-architecture image. That will probably
	// change in the future.
```
, we have explicitly decided not to deal with `ImageListSelection`.